apps/web/vibes/soul/components/compare-card/index.tsx

```diff
@@ -68,28 +68,30 @@ export const CompareCard = function CompareCard({
       <div className="mb-2 flex flex-col gap-1">
         <h3 className="flex flex-col flex-wrap justify-between gap-1 text-sm font-semibold @sm:pt-3 @4xl:flex-row">
           {name && <span className="line-clamp-2">{name}</span>}
-          {subtitle && <span className="font-normal text-contrast-400">{subtitle}</span>}
+          {subtitle != null && (
+            <span className="font-normal text-contrast-400">{subtitle}</span>
+          )}
```

Comment:
Just want to double-check that for some of these the intent wasn't to hide the entire element if the string was empty? i.e., do we actually want to do `subtitle && ...` but ignore the ESLint rule? Or perhaps be explicit and do `subtitle != null && subtitle !== '' && ...`. Same goes for other components were if the prop is an empty string we don't want to render anything